package.json

```diff
@@ -31,27 +31,27 @@
     "html"
   ],
   "devDependencies": {
-    "@babel/core": "^7.9.6",
-    "@babel/preset-env": "^7.9.6",
+    "@babel/core": "^7.10.4",
+    "@babel/preset-env": "^7.10.4",
     "@markedjs/html-differ": "^3.0.2",
     "cheerio": "^1.0.0-rc.3",
-    "commonmark": "0.29.x",
-    "eslint": "^7.0.0",
+    "commonmark": "0.29.1",
```

Comment:
Did you intend to change `commonmark`? I noticed that `markdown` and `markdown-it` still use the `x` syntax.

Reply:
Ya since we are using `0.29.1` specs I figured we should probably pin it and update this dependency when we update the specs.